src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11692,7 +11693,7 @@ void Player_Action_8084B1D8(Player* this, PlayState* play) {
     }
 
     if ((this->csAction != PLAYER_CSACTION_NONE) || (this->unk_6AD == 0) || (this->unk_6AD >= 4) ||
-        func_80833B54(this) || (this->unk_664 != NULL) || !func_8083AD4C(play, this) ||
+        func_80833B54(this) || (this->unk_664 != NULL) || (func_8083AD4C(play, this) == CAM_MODE_NORMAL) ||
```

Comment:
I'm still not convinced about this but I guess it'll become clearer when player has more docs :tm: